src/xenia/kernel/xboxkrnl/xboxkrnl_module.cc

```diff
@@ -189,6 +189,14 @@ XboxkrnlModule::XboxkrnlModule(Emulator* emulator, KernelState* kernel_state)
                                        ordinals::XexExecutableModuleHandle,
                                        ppXexExecutableModuleHandle);
 
+  // ExLoadedImageName (char*)
+  // The full path to loaded image/xex including its name.
```

Comment:
Probably need a comment confirming that the full kExLoadedImageNameSize (disregarding the actual length) is always allocated in the official kernel.